R/annotation.r

```diff
@@ -29,6 +29,11 @@
 #'   colour = "red", size = 1.5)
 #'
 #' p + annotate("text", x = 2:3, y = 20:21, label = c("my label", "label 2"))
+#'
+#' p + annotate("text", x = 4, y = 25, label = "italic(R) ^ 2 == 0.75",
+#'   parse = TRUE)
+#' p + annotate("text", x = 4, y = 25,
```

Comment:
It shows, that plain text can be mixed with plotmath expressions.

Using plotmath alone, it would not be possible to trim the leading zero of the r-squared statistic. In Psychology (according to the publication manual of the American Psychological Association) values that can't exceed 1 are typically printed without leading zero.